bash/cmd-line-searches/howmuch

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+#=======================================================================================================
+#title           :howmuch
+#description     :Lists sizes of all files in specified directory human readably, and sorted descending
+#author          :dharvey-consbio
+#usage           :see https://github.com/consbio/cookbook/tree/master/bash/cmd-line-searches/#howmuch
+#=======================================================================================================
+
+export GLOBIGNORE=".:.."
+
+if [[ -z $1 ]] ; then
+  export searchin=.
```

Comment:
Also, `export` isn't necessary unless you need the variable to exist outside the scope of the script.

Reply:
Using `export` means that the variable will be accessible to sub-scripts. It will not be accessible to the calling shell unless called like `. howmuch /usr/bin`.

I agree it's not strictly necessary here, but I typically use `export` as a best practice, because I have run into very difficult issues arising from variables I thought were set not being set.